Dockerfile

```diff
@@ -86,13 +86,14 @@ ARG BIN_SUFFIX
 # which flavor (production or testing). The actual binary name in the
 # image is going to be the same, to avoid unnecessary deployment
 # differences.
-RUN make VERSION=${VERSION} pmem-csi-driver${BIN_SUFFIX} pmem-vgm${BIN_SUFFIX} pmem-ns-init${BIN_SUFFIX} && \
+RUN set -x && \
```

Comment:
@pohly Is `set -x` is intentional here? I guess by default all RUN statements are echoed by `docker build`.

Reply:
@pohly Ignore my earlier comment. Here I see adding `set -x` seems useful for debugging individual commands inside RUN

Reply:
Exactly. In the output without `-x` it was impossible to see which command actually caused Go to fetch dependencies.